shimmingtoolbox/cli/maths.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+
+import click
+import os
+import nibabel as nib
+import numpy as np
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+AXES = ['0', '1', '2', '3', '4']
```

Comment:
in 99% of the case, axis will be 3 (time). I suggest to put it in default. Same comment for the math CLI